.semaphore/semaphore.yml

```diff
@@ -34,23 +34,35 @@ global_job_config:
         - make store-test-results-to-semaphore
 
 blocks:
-  - name: "Build and Test"
+  - name: "Build JARs and Unit Test"
     dependencies: []
     task:
       prologue:
         commands:
           - sem-version java 21
+      epilogue:
+        always:
+          commands:
+            - emit-sonarqube-data --run_only_sonar_scan
```

Comment:
We split the "Build and Test" block into two so that we can configure `epilogue.always` on just "Build JARs and Unit Test"